### DISCUSSION TITLE
Shell

### DISCUSSION THREAD

**Discussion by enrealsec** [cat-milk/Anime-Girls-Holding-Programming-Books#371] [Ideas]
Hi, I was looking here and noticed that it doesn't have a shell image.

**Comment by Ikpechimisheal** [comment]
How would one she'll an image

**Comment by Asday** [comment]
> Hi, I was looking here and noticed that it doesn't have a shell image.

Sounds like a personal problem.

https://krita.org/en/

**Reply by Ikpechimisheal** [reply]
Sure

**Comment by maratbekovaidar** [comment]
Good
